pyseir/icu/utils.py

```diff
@@ -28,10 +28,13 @@ def calculate_case_based_weights() -> dict:
     last_month_cum_cases.name = "summed_cases"
 
     df = last_month_cum_cases.reset_index().dropna()
-    df["state_fips"] = df["fips"].str.slice(0, 2)  # Apply State Labels to Groupby
+    # Example location_id value = 'iso1:us#iso2:us-ak#fips:02013'
+    df["state_location_id"] = df["location_id"].str.split("#").str[1]  # 'iso2:us-ak'
```

Comment:
What do you want to happen for MSAs, that don't have a single state?

Reply:
My first inclination is to sum up all the estimated ICU numbers from the county level. That breaks the hierarchy a little though. So you would disaggregate from states to counties and then reaggregate back to MSAs. There is an edge case where one state has actuals while the other has estimates that will result in a blended estimate, but I think that's the best we can do. Open to suggestions. Do you have any ideas?